packages/inspector/src/components/Renderer/AxisHelper/AxisHelper.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { useEffect, useRef } from 'react';
+import * as BABYLON from '@babylonjs/core';
+import { useSdk } from '../../../hooks/sdk/useSdk';
+import './AxisHelper.css';
+
+const AxisHelper: React.FC = () => {
+  const sdk = useSdk();
+  const containerRef = useRef<HTMLDivElement>(null);
+  const canvasRef = useRef<HTMLCanvasElement>(null);
+
+  useEffect(() => {
+    if (!sdk || !canvasRef.current) return;
+
+    const canvas = canvasRef.current;
+    const engine = new BABYLON.Engine(canvas, true, {
+      preserveDrawingBuffer: true,
+      stencil: true,
+      antialias: true,
+    });
```

Comment:
is it required to create a new engine and scene?

what do you think about moving this Babylon logic to a new file: `packages/inspector/src/lib/babylon/setup/axisHelper` similar to: https://github.com/decentraland/creator-hub/blob/main/packages/inspector/src/lib/babylon/setup/setup.ts?